include/rtc/h265packetizationhandler.hpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2020 Filip Klembara (in2core)
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef RTC_H265_PACKETIZATION_HANDLER_H
+#define RTC_H265_PACKETIZATION_HANDLER_H
+
+#if RTC_ENABLE_MEDIA
+
+#include "h265rtppacketizer.hpp"
+#include "mediachainablehandler.hpp"
+#include "h265nalunit.hpp"
+
+namespace rtc {
+
+/// Handler for H265 packetization
+class RTC_CPP_EXPORT H265PacketizationHandler final : public MediaChainableHandler {
```

Comment:
I'm currently refactoring media handlers to simplify the design in https://github.com/paullouisageneau/libdatachannel/pull/929. One of the main changes will be that packetization handlers are not necessary anymore.

This could be merged before (in which case I'll refactor it too) or after, I'm fine with both.

Reply:
Hi @paullouisageneau , thanks for your review :) I just rebase today, and didn't merge https://github.com/paullouisageneau/libdatachannel/pull/929 since it's still open.